tony-history-server/app/controllers/JobsConfigPageController.java

```diff
@@ -0,0 +1,47 @@
+package controllers;
+
+import com.typesafe.config.Config;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import models.JobConfig;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import play.Logger;
+import play.Logger.ALogger;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+import static utils.HdfsUtils.*;
+import static utils.ParserUtils.*;
+import static utils.SecurityUtils.*;
+
+
+public class JobsConfigPageController extends Controller {
+  private static final ALogger LOG = Logger.of(JobsConfigPageController.class);
+  private final Config config;
+
+  @Inject
+  public JobsConfigPageController(Config config) {
+    this.config = config;
+    getInstance(config);
+  }
+
+  public Result index(String jobId) {
+    List<JobConfig> listOfConfigs = new ArrayList<>();
+    FileSystem myFs = getFs();
+    String tonyHistoryFolder = config.getString("tony.historyFolder");
```

Comment:
make tony.historyFolder constant

Reply:
Something like this: https://github.com/linkedin/TonY/blob/master/tony-core/src/main/java/com/linkedin/tony/Constants.java?

Reply:
yup

Reply:
Hmm so in that case can I put `Constants` in utils?

Reply:
Actually, on a second thought, I don't think we should put it in Constants file, as `tony.historyFolder` is different for dev/prod mode.

Reply:
`tony.historyFolder` is the name of the parameter, which is constant. The actual value (which will be different in dev/prod) comes from the `application.conf` file (or later from `$TONY_CONF_DIR/tony-site.xml`).